src/test/groovy/org/araneforseti/boundary/TestUtil.groovy

```diff
@@ -45,6 +50,29 @@ class TestUtil {
         return found
     }
 
+    static boolean scenario_messages_contains(List<String> values, Field field) {
+        List<Boolean> results = []
+        values.each { value ->
+            results << scenario_messages_contains(value, field)
+        }
+        !results.contains(false)
+    }
+
+    static boolean scenario_messages_contains(List<String> values, List<Field> fields) {
+        List<Boolean> results = []
+        fields.each { field ->
+            results << scenario_messages_contains(values, field)
+        }
+        !results.contains(false)
+    }
+
+    static void assert_messageConfiguration_is_used_for_scenarios(MessageConfiguration messageConfiguration, Field optional, Field required) {
```

Comment:
Don't think this is the right place for that due to how I prefer separation of concerns between useful encapsulations vs actual assertions